plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -497,7 +495,7 @@ public JSONArray getPluginDependencyJsonArray(Plugin plugin, JSONObject ucJson)
 
             //not all plugin Manifests contain the Plugin-Dependencies field
             if (StringUtils.isEmpty(dependencyString)) {
-                System.out.println(plugin.getName() + " has no dependencies");
+                log("\n" + plugin.getName() + " has no dependencies");
```

Comment:
Do you need System.LINE_SEPARATOR here? (not sure if java handles \n on different OS sending the right separator?